Android/src/org/droidplanner/android/dialogs/UVCDialog.java

```diff
@@ -0,0 +1,209 @@
+package org.droidplanner.android.dialogs;
+
+import android.app.Activity;
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.app.DialogFragment;
+import android.content.Context;
+import android.content.DialogInterface;
+import android.hardware.usb.UsbDevice;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.BaseAdapter;
+import android.widget.Button;
+import android.widget.CheckedTextView;
+import android.widget.Spinner;
+
+import com.serenegiant.usb.DeviceFilter;
+import com.serenegiant.usb.USBMonitor;
+
+import org.droidplanner.android.R;
+
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+public class UVCDialog extends DialogFragment {
+
+    private static final String TAG = UVCDialog.class.getSimpleName();
+
+
+    protected USBMonitor mUSBMonitor;
+    private Spinner mSpinner;
+    private DeviceListAdapter mDeviceListAdapter;
+
+    /**
+     * Helper method
+     * @param parent FragmentActivity
+     * @return
+     */
+    public static UVCDialog showDialog(final Activity parent) {
+        UVCDialog dialog = newInstance();
+        try {
+            dialog.show(parent.getFragmentManager(), TAG);
+        } catch (final IllegalStateException e) {
+            dialog = null;
+        }
+        return dialog;
+    }
+
+    /**
+     * Helper method
+     * @param parent FragmentActivity
+     * @return
+     */
+    public static UVCDialog showDialog(final Activity parent, USBMonitor mUSBMonitor) {
+        UVCDialog dialog = newInstance();
+        dialog.mUSBMonitor = mUSBMonitor;
+        try {
+            dialog.show(parent.getFragmentManager(), TAG);
+        } catch (final IllegalStateException e) {
+            dialog = null;
+        }
+        return dialog;
+    }
+
+    public static UVCDialog newInstance() {
+        final UVCDialog dialog = new UVCDialog();
+        final Bundle args = new Bundle();
+        dialog.setArguments(args);
```

Comment:
@guiseco what's the `Bundle args` for here, since no arguments is being added to it?